push-changes.sh

```diff
@@ -4,7 +4,7 @@ set -e
 if [[ `git status --porcelain` ]]; then
   git add locales/*
   git commit -m "auto update translation"
-  git push origin HEAD:master
+  git push $UPSTREAM_REPO HEAD:master
```

Comment:
Is it worth having a default fallback for this?

Reply:
I made it a variable rather than hard-coding specifically to avoid having a default value - otherwise I could see myself running this on my local machine at some point and pushing some testing to the actual repo.

>